apps/trading/components/fees-container/table.tsx

```diff
@@ -3,21 +3,49 @@ import type { ReactNode } from 'react';
 
 const cellClass = 'px-4 py-2 text-xs font-normal text-left last:text-right';
 
-export const Th = ({ children }: { children?: ReactNode }) => {
+export const Th = ({
+  children,
+  testId,
+}: {
+  children?: ReactNode;
+  testId?: string;
```

Comment:
I would just type these props as `HTMLProps<HTMLTableHeaderCellElement>` and pass data-testid directly. Same for the other table element components

Reply:
Fixed